owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -112,11 +112,24 @@ abstract class DrawerActivity : ToolbarActivity() {
         getDrawerLayout()?.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
         getNavView()?.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
 
-        // Set background header image and logo, if any
+        // Set background header image, if any
         if (resources.getBoolean(R.bool.use_drawer_background_header)) {
             getDrawerHeaderBackground()?.setImageResource(R.drawable.drawer_header_background)
         }
-        if (resources.getBoolean(R.bool.use_drawer_logo)) {
+
+        // Set logo and text for drawer link, if any
+        if (isDrawerLinkEnabled()) {
+            getDrawerLinkIcon()?.apply {
+                isVisible = true
+                setImageResource(R.drawable.drawer_logo)
+                setOnClickListener { openDrawerLink() }
+            }
+            getDrawerLinkText()?.apply {
+                isVisible = true
+                text = resources.getString(R.string.drawer_link_label)
```

Comment:
Icon and Text will be always the same: `R.drawable.drawer_logo` and `R.string.drawer_link_label`, so I think we could add them to the XML and make them visible here. Am I missing something?